### PR TITLE
Create a feature request issue template

### LINKED ISSUE #27: Edit Issue Template for Feature Requests
## Actual Behavior

This issue template sucks for new features.

## Expected Behavior

This issue template is relevant for new features.

## Steps to Reproduce

1. Navigate to https://github.com/585-software/rocdev/issues/new.

## Relevant Specifications (OS, Hardware, etc.)

N/A

### PR DESCRIPTION
**Pull request by kylemacey**
Creates an issue template for requesting features. Will be accessible at https://github.com/585-software/rocdev/issues/new?template=feature.md once merged. 

Closes #27 

cc:\ @geowa4 

### TODO

- [x] Link from README.md or CONTRIBUTING.md (looking for feedback here)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Feature description
+
+Describe in full detail the behavior that you would like implemented. Use screenshots or mock-ups if applicable.
```
Comment (geowa4) [inline .github/ISSUE_TEMPLATE/feature.md]: Can you put each sentence on its own line? It makes it easier to rearrange and delete them while also making diffs easier to read.

File: .github/ISSUE_TEMPLATE/feature.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+### Feature description
```
Comment (geowa4) [inline .github/ISSUE_TEMPLATE/feature.md]: Can you capitalize the major words in headings?

**Review by geowa4 — CHANGES_REQUESTED** [review]
You're showing me some omissions in my CONTRIBUTING.md.  I'm making a bug for that.

**Comment by geowa4** [comment]
Regarding where to link to this, I think it would be best as a sibling to [Getting Started](https://github.com/585-software/rocdev#getting-started) in README.md.

**Comment by kylemacey** [comment]
@geowa4 Updated!

**Review by geowa4 — APPROVED** [review]

**Comment by geowa4** [comment]
Thanks, @kylemacey. I'm working on #9 now. Once I'm done with that I was going to do #38, but I just realized that's another issue you or someone else could do to get a Hacktoberfest shirt.

**Pull request merged**